### PR TITLE
added jupyter notebook converter

### PR DESCRIPTION
**Pull request by AlexFreik**
I placed option to open jupyter notebook inside navbar menu button
Screenshot 2022-09-26 at 12 56 34

### CODE REVIEW

File: frontend/components/frames/UiNotebookViewerFrame.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+	// useEffect(() => {/* empty */}, [awu.selectedFrame]);
+
+	const ipynb = graphToNotebook(frame.graph, frame.executer);
+	return (
+		<div style={{ overflow: 'scroll', height: '100%' }}>
+			<IpynbRenderer
+				ipynb={ipynb}
+				syntaxTheme="xonokai"
+				language="python"
```
Comment (roman-timo) [inline frontend/components/frames/UiNotebookViewerFrame.tsx:22]: language python?? )

File: frontend/src/frames/NotebookViewerFrame.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+	public getName() {
+		return 'code: ' + this.graph.name;
+	}
+
+	public static getType(): string {
+		return 'NotebookViewerFrame';
+	}
+
+	public getType(): string {
```
Comment (roman-timo) [inline frontend/src/frames/NotebookViewerFrame.ts:16]: is it a good thing to have multiple methods for same function..? Maybe just rename it for simplicity?

File: frontend/src/utils/notebook_utils.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import { ScriptNode } from '../graph/nodes/ScriptNode';
+import { ResultPart } from '../kernels/execution';
+
+export function graphToNotebook(graph: AtlasGraph, executer: JuliaExecuter): any {
+	const cells = graph.nodes
+		.filter((node) => node instanceof ScriptNode)
+		.map((node) => nodeToCell(node as ScriptNode, graph, executer))
+		.sort((a, b) => {
+			const left = a.execution_count === null ? 0 : a.execution_count;
+			const right = b.execution_count === null ? 0 : b.execution_count;
```
Comment (roman-timo) [inline frontend/src/utils/notebook_utils.ts:13]: this could be just 
`Number(a.execution_count)`

**Review by roman-timo — CHANGES_REQUESTED** [review]

**Comment by roman-timo** [comment]
just why it is black...??)

File: frontend/components/frames/UiNotebookViewerFrame.tsx (hunk 1 above)
Comment (AlexFreik) [reply frontend/components/frames/UiNotebookViewerFrame.tsx:22]: the only options are "r" and "python"

File: frontend/src/frames/NotebookViewerFrame.ts (hunk 2 above)
Comment (AlexFreik) [reply frontend/src/frames/NotebookViewerFrame.ts:16]: one is static, and other one is not. It is to be able to do

```ts
if (AtlasGraphFrame.getType() == frame.getType()) do something;
```

**Comment by AlexFreik** [comment]
> just why it is black...??)

The other option is solarizedl and it is shit)

Screenshot 2022-09-27 at 14 43 31

**Comment by AlexFreik** [comment]
Updated the theme

Screenshot 2022-09-27 at 15 06 45

File: frontend/src/utils/notebook_utils.ts (hunk 3 above)
Comment (AlexFreik) [reply frontend/src/utils/notebook_utils.ts]: Ahah, indeed) fixed

**Review by roman-timo — APPROVED** [review]
Good to go

**Review by roman-timo — APPROVED** [review]
Good

**Pull request merged**